teams/teams-ps/teams/Grant-CsGroupPolicyPackageAssignment.md

```diff
@@ -0,0 +1,109 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+title: Grant-CsGroupPolicyPackageAssignment
+author: icchan
+ms.author: icchan
+manager: amitar
+online version: https://docs.microsoft.com/powershell/module/teams/grant-csgrouppolicypackageassignment
+schema: 2.0.0
+---
+
+# Grant-CsGroupPolicyPackageAssignment
+
+## SYNOPSIS
+
+**Note:** This cmdlet is currently only available in private preview in the pre-release Teams PowerShell module.
+
+This cmdlet applies a policy package to a group in a tenant.
+
+## SYNTAX
+
+```
+Grant-CsUserPolicyPackage -GroupId <String> -PackageName <String> [-Rank <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet applies a policy package to a group in a tenant. Provide a group id to assign the package with all the associated policies. The available policy packages and their definitions can be found by running Get-CsPolicyPackage. Rank can be optionally specified to determine which policies in a package will be applied to the user in case they belong to two or more groups. If rank is not specified, the group's package assignment will be ranked last. If a user was directly assigned a package, direct assignment takes precedence over group assignment. For more information on rank and group policy assignments, please review [this section](New-CsGroupPolicyAssignment.md#description). For more information on policy packages, please review https://docs.microsoft.com/en-us/MicrosoftTeams/manage-policy-packages
```

Comment:
```suggestion
This cmdlet applies a policy package to a group in a tenant. Provide a group id to assign the package with all the associated policies. The available policy packages and their definitions can be found by running Get-CsPolicyPackage. Rank can be optionally specified to determine which policies in a package will be applied to the user in case they belong to two or more groups. If rank is not specified, the group's package assignment will be ranked last. If a user was directly assigned a package, direct assignment takes precedence over group assignment. For more information on rank and group policy assignments, please review [this section](New-CsGroupPolicyAssignment.md#description). For more information on policy packages, please review https://docs.microsoft.com/MicrosoftTeams/manage-policy-packages.
```

Reply:
Fixed, thanks